src/NosCore.Controllers/CharacterScreenPacketController.cs

```diff
@@ -369,6 +369,7 @@ public void SelectCharacter(SelectPacket selectPacket)
                 character.MapInstance = _mapInstanceAccessService.GetMapInstance(character.MapInstanceId);
                 character.PositionX = character.MapX;
                 character.PositionY = character.MapY;
+                character.Direction = 2;
```

Comment:
Why 2 ? Should be last dir

Reply:
Last dir? Dir is saved too? Anyways, official doesn't save the dir when logging out.

Reply:
are you sure about that ?

Reply:
I checked it some hours ago, only to make sure I will go and check it now.

Reply:
Yeah, it's by default 2